### DISCUSSION TITLE
Please show us an example of composing huggingface models

### DISCUSSION THREAD

**Discussion by ishaanpo** [lucidrains/CALM-pytorch#3] [Q&A]
Most of us all use huggingface models on the daily, so it would be nice to be able to compose these models. Thanks!

**Comment by lucidrains** [comment]
yea i will

**Reply by ishaanpo** [reply]
thanks!

**Comment by RRaphaell** [comment]
any updates on that?

**Comment by ayushayush591** [comment]
Are you planning to do that?
